pages/Iceland.html

```diff
@@ -0,0 +1,153 @@
+                    </div>
+                </div>
+                <div class="image-container">
+                    <img src="../img/Iceland/puffin_iceland.jpg" alt="Puffin image" class="img front-img">
+                    <div class="image-text">
+                        <h1>Puffin</h1>
+                        <p>Islanda oferă un habitat natural spectaculos pentru Puffin, păsările colorate și adorable. Observarea Puffin-ilor în sălbăticie este o experiență unică și autentică. Înspre sfârșitul verii, insula este locul perfect pentru a admira aceste păsări migratoare. Puffin-ii din Islanda sunt considerați un simbol al biodiversității și conservă</p>
+                    </div>
+                </div>
+                <div class="image-container">
+                    <img src="../img/Iceland/iceland-whale.jpg" alt="Whale image" class="img front-img">
+                    <div class="image-text">
+                        <h1>Balenă</h1>
+                        <p>Islanda este unul dintre cele mai bune locuri din lume pentru observarea balenelor în sălbăticie. Aici puteți întâlni diverse specii de balene, inclusiv balene cu cocoș, balene cu cioc lung și balene cu cocoașe albastre. Există numeroase excursii cu barca care vă duc în apropierea acestor creaturi impresionante. Observarea balenelor în Islanda oferă o experiență unică de conectare cu natura și de înțelegere a vieții marine.</p>
+                    </div>
+                </div>
+                <div class="image-container">
+                    <img src="../img/Iceland/glaciar_iceland.jpg" alt="Iceberg image" class="img front-img">
+                    <div class="image-text">
+                    <h1>Iceberg</h1>
+                <p>Islanda este faimoasă pentru glaciarele sale masive și impresionante. Cele mai cunoscute glaciare includ Vatnajökull, cel mai mare din Europa. Aceste minuni naturale oferă peisaje dramatice și aventuri unice. Turiștii pot explora ghețarii pe jos sau în călătorii organizate, deci, hai cu noi!.</p>
+                </div>
+            </div>
+            <div class="image-container">
+                <img src="../img/Iceland/cascade-iceland.jpg" alt="Goðafoss image" class="img front-img">
+                <div class="image-text">
+                    <h1>Goðafoss</h1>
+                    <p>Islanda este faimoasă pentru glaciarele sale masive și impresionante. Cele mai cunoscute glaciare includ Vatnajökull, cel mai mare din Europa. Aceste minuni naturale oferă peisaje dramatice și aventuri unice. Turiștii pot explora ghețarii pe jos sau în călătorii organizate, deci, hai cu noi!.</p>
+                </div>
+            </div>
+            <div class="go-to-exeditiond">
+                <a href="../index.html">Vezi toate expedițiile</a>
```

Comment:
Aici puteai pune link care să ducă direct la secțiunea de expediții.

```suggestion
                <a href="../index.html#expeditii">Vezi toate expedițiile</a>
```